### PR TITLE
Using kubectl packages to run "kubectl apply", without requiring

### PR DESCRIPTION
**Pull request by aleksandrmelnikov**
the kubectl binary to be present.

### CODE REVIEW

**Comment by rushtehrani** [comment]
@aleksandrmelnikov does this allow for passing options to `kubectl get`? At first glance it doesn’t look like it.

File: util/kubectl.go [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+	f := cmdutil.NewFactory(matchVersionKubeConfigFlags)
+
+	ioStreams := genericclioptions.IOStreams{
+		In:     os.Stdin,
+		Out:    out,
+		ErrOut: errOut,
+	}
+	cmd := get.NewCmdGet("kubectl", f, ioStreams)
+	args := []string{resource, resourceName}
```
Comment (aleksandrmelnikov) [inline util/kubectl.go]: @rushtehrani Right here is where you would use "kubectl get".
- resource = "pod/pods | namespace"
- resourceName = "podName | namespaceName"

Comment (aleksandrmelnikov) [reply util/kubectl.go]: Wait, what do you mean by options?

Comment (rushtehrani) [reply util/kubectl.go]: For example `-o` in the scenario we discussed earlier:
```
kubectl get svc istio-ingressgateway -n istio-system -o jsonpath='{.status.loadBalancer.ingress[0].ip}'
```

**Pull request merged**